docs/pages/eas/json.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: eas.json
+description: Learn about available properties for EAS Build and EAS Submit to configure and override their default behavior from within your project.
+maxHeadingDepth: 4
+---
+
+import { EasJsonPropertiesTable } from '~/components/plugins/EasJsonPropertiesTable';
+import { Collapsible } from '~/ui/components/Collapsible';
+
+import commonSchema from '~/public/static/schemas/unversioned/eas-json-build-common-schema.js';
+import androidSchema from '~/public/static/schemas/unversioned/eas-json-build-android-schema.js';
+import iosSchema from '~/public/static/schemas/unversioned/eas-json-build-ios-schema.js';
+import submitAndroidSchema from '~/public/static/schemas/unversioned/eas-json-submit-android-schema.js';
+import submitIosSchema from '~/public/static/schemas/unversioned/eas-json-submit-ios-schema.js';
+
+**eas.json** is the configuration file for EAS CLI and services. It is located at the root of a project next to the **package.json**. You can find the complete reference of all available schema properties for [EAS Build](/build/introduction) (`build`) and [EAS Submit](/submit/introduction) (`submit`) on this page.
+
+## EAS Build
```

Comment:
we could add a sentence here that explains these properties exist under the `build` key and have a collapsible example that people can look at. it's useful to have something like this on the page so people can quickly see what this table translates to in their project. we could reuse the eas.json example from http://docs.expo.dev-pr-24491.s3-website-us-east-1.amazonaws.com/build/eas-json/

Reply:
Added the description and the example

<img width="901" alt="CleanShot 2023-09-24 at 16 54 39@2x" src="https://github.com/expo/expo/assets/10234615/63dc930a-3521-4999-a96c-c7d78afa07eb">